tests/test_interface.py

```diff
@@ -335,7 +335,7 @@ def test_interface_add_garbage(self):
 
     @mock.patch.object(Payment, 'deserialize')
     @mock.patch.object(Payment, 'serialize', return_value=CC_RETURN)
-    @mock.patch.object(app_db, 'session', autospec=True)
+    @mock.patch.object(app_db, 'session')
```

Comment:
yeah, good call. i just struck them from existing tests, which also required removing the `mock.ANY` params used to mock the `self` param. Didn't add anything to the tests and I know in my case, it was merely a by-product of experimenting with different combinations of mocking to see how it all worked.